docs/Set-PSReadLineOption.md

```diff
@@ -612,6 +615,28 @@ Accept pipeline input: false
 Accept wildcard characters: false
 ```
 
+### -PredictionStyle
```

Comment:
>PredictionStyle [](start = 5, length = 15)

This parameter suggests something other than rendering, e.g. prediction from history or some other magic. How about `PredictionViewStyle`?

Reply:
Think about it more, I do want to use it beyond rendering. The idea is to have `Concise` (can be named `Simple` as you suggested) and `List` options.
- When `Simple` is in use, PSReadLine will use history as the only source for the inline suggestion. The suggestion text will be rendered inline just like what the prototype does today.
- When `List` is in use, besides history, PSReadLine will also call into a new API in `S.M.A`, which will call into a potential prediction plugin, to get suggestions. The suggestion results (up to 2 results from history + up to 4 results from the API) will be rendered as a drop down list, which can be navigated by <kbd>UpArrow</kbd> and <kbd>DownArrow</kbd> and selected by <kbd>Enter</kbd> or <kbd>RightArrow</kbd>.

The 2nd part is still in design and will raise privacy concerns as PSReadLine will be sending the user input to somewhere (in the cloud?). But it should clarify that this option does suggest something more than rendering.

Reply:
Do you want to mix the display style and prediction source?  Seems like those could be two different settings.  Maybe I'd like to see a list of predictions from just my PSRL history?  Or maybe I only want to see the simple/single view but from an alternate source (not yet implemented)?

Reply:
When the suggestion is from a different source, there is no guarantee the suggestion can be rendered inline (simple/single view), because the returned results may not be prefixed with what the user have typed on the console. But maybe it's better to have 2 separate settings anyways, it's more clear that way.